docs/eventing/broker/kafka-broker.md

```diff
@@ -19,6 +19,7 @@ Notable features are:
 ## Prerequisites
 
 [Knative Eventing installation](./../../install/any-kubernetes-cluster.md#installing-the-eventing-component).
+[Apache Kafka cluster installation](https://strimzi.io/quickstarts/)
```

Comment:
Thanks much! I wonder if we should point to the existing doc that has more details?
https://knative.dev/docs/eventing/samples/kafka/index.html

@matzew thoughts?

Reply:
And then if we want to we can add more options here? Just so there's one place where we guide on how to install kafka instead of direct linking?

Reply:
IMO that page is obsolete (Strimzi version is v0.16 vs current version v0.21) and it doesn't add much to the Strimzi quickstart page.
We should just link to the Strimzi quickstart page since we will get the benefit of their simplified installation work and people install a supported version of it (just in case someone finds a bug while installing it).

Reply:
> We should just link to the Strimzi quickstart page 

agreed!